### PR TITLE
AM - Changes on filters of the Library page

### PR DESCRIPTION
**Pull request by KevinDurand974**
From: https://github.com/blockframes/blockframes/issues/8680

- [x] Need migration for changing certifications in db ?
   <del>`No, this will be done manually`</del>
   `Done with migration after reflection`
   
   
Expect 1 bug for some movies who will display Qualifications / Certifications, like :
```
, EOF (French Speaking Films), European Qualification
```
_This will be corrected automatically when the anonymous database will be recreated after they manually edit the prod version._

### CODE REVIEW

**Review by phpgeekfr — COMMENTED** [review]
this is the reason why you changed the excel templates ? 
image

File: libs/model/src/lib/static/static-model.ts [hunk 1]
```diff
@@ -57,10 +57,7 @@ export const offerStatus = {
 export const negotiationStatus = ['pending', 'accepted', 'declined'] as const;
 
 export const certifications = {
-  artEssai: 'Art & Essai',
-  eof: 'EOF',
```
Comment (phpgeekfr) [inline libs/model/src/lib/static/static-model.ts:61]: Camille said `Add (French Speaking Films) next to EOF` but other ceritifcations like awardedFilm aListCast have to be removed ?

File: libs/model/src/lib/static/static-model.ts [hunk 2]
```diff
@@ -57,10 +57,7 @@ export const offerStatus = {
...
-  awardedFilm: 'Awarded Film',
-  aListCast: 'A-list Cast',
+  eof: 'EOF (French Speaking Films)',
   europeanQualification: 'European Qualification'
 } as const
```
Comment (phpgeekfr) [inline libs/model/src/lib/static/static-model.ts:62]: please remove export on this 
```
export type StaticModel = typeof staticModel;
```

and thoses two unused lines
```
export type GetLabel<S extends Scope> = StaticModel[S][GetKeys<S>]
export type GetCode<S extends Scope> = GetKeys<S> | GetLabel<S>;
```

**Review by phpgeekfr — COMMENTED** [review]
- [ ] something with responsive is broken.
on develop (and this branch)
image

on staging
image


If you can have a look ..

File: libs/model/src/lib/static/static-model.ts (hunk 1 above)
Comment (KevinDurand974) [reply libs/model/src/lib/static/static-model.ts:61]: These ones are not used anymore, I checked before doing this

**Comment by KevinDurand974** [comment]
> this is the reason why you changed the excel templates ? image

Yes!

**Comment by phpgeekfr** [comment]
@cams-goussale it now takes a bit too much place 
image

Is it ok or can we shrink some text buttons ?

**Comment by cams-goussale** [comment]
> Is it ok or can we shrink some text buttons ?

there should be the scroll bar to see the rest of the filters. It's ok

**Comment by phpgeekfr** [comment]
- [ ] if you have time, update this migration to perform needed updates `libs/devops/src/lib/firestoreMigrations/0084.ts`

File: libs/movie/src/lib/movie/form/search.form.ts [hunk 3]
```diff
@@ -151,6 +157,8 @@ export class MovieSearchForm extends FormEntity<MovieSearchControl> {
   get hitsPerPage() { return this.get('hitsPerPage'); }
   get contentType() { return this.get('contentType'); }
   get runningTime() { return this.get('runningTime'); }
+  get festivals() { return this.get('festivals'); }
+  get certifications() { return this.get('certifications') }
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/search.form.ts:161]: for consistency
```suggestion
  get certifications() { return this.get('certifications'); }
```

File: libs/movie/src/lib/movie/form/search.form.ts [hunk 4]
```diff
@@ -168,6 +176,7 @@ export class MovieSearchForm extends FormEntity<MovieSearchControl> {
       this.minBudget?.value === 0 &&
       this.minReleaseYear.value === 0 &&
       this.sellers?.value.length === 0 &&
+      this.festivals.value?.length === 0 &&
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/search.form.ts:179]: why no `certifications` here.
Also, whole `isEmpty` method looks like unused

**Review by phpgeekfr — APPROVED** [review]

**Pull request merged**